### ISSUE TITLE
Peertube Embed Api addEventListener doesn't work in iphone, using safari or webview in app

### ISSUE THREAD

**Issue by Will-lopesz** [Chocobozzz/PeerTube#6364]
### Describe the current behavior

Hello, I noticed an issue created in 2022 (#5389) that describes exactly what I'm experiencing: the eventListener 'playbackStatusUpdate' simply doesn't work on iOS.

Currently, we're using an instance of PeerTube in a Docker container, running version v6.1.0. We recently updated it, but the issue persists. We consume the videos through the embed API in a Vue.js project.

In an attempt to locate the problem, I started a new Vue.js project and used a video from your PeerTube instance (peertube.tv), and a video from our instance, both playing simultaneously on the same screen. Ours doesn't capture the update events, whereas on peertube.tv it works perfectly. Additionally, on other devices that don't run iOS, ours also works perfectly.

I would appreciate some help in resolving this issue. I'm available to provide any necessary information; we're unsure what could be causing this. Do you have any ideas? Has anyone else experienced this while running an instance on Docker?

### Steps to reproduce

1. To reproduce the issue, start a web project and use our embed link in an iframe.
2. Add an event listener for playbackStatusUpdate.
3. Run the project and open it on an iOS device; note that the values will never be updated.

```
<template>
    <div id="video-lesson" class="flex justify-center">
        <div class="general-container">
            <iframe id="peertube" title="Arthur's Seat, Edinburgh, at sunset 22nd April 2024" src="https://peertube.tv/videos/embed/46a6159f-2d52-43c4-a687-f4b9ae0b5402?api=1" frameborder="0" allowfullscreen="" sandbox="allow-same-origin allow-scripts allow-popups"> </iframe>
        </div>
        <div style="margin-bottom: 70px">
            {{ playback }}
        </div>
        <div class="separator"></div>
        <div class="general-container">
            <iframe id="imlabs" title="Nosso video da nossa instancia do PeerTube" src="https://a.video.imlabs.com.br/videos/embed/d76b8444-8d52-41df-ac61-732861c41687?api=1" frameborder="0" allowfullscreen="" sandbox="allow-same-origin allow-scripts allow-popups"> </iframe>
        </div>
        <div>
            {{ playback2 }}
        </div>
    </div>
</template>

<script>
import { PeerTubePlayer } from "@peertube/embed-api";

export default {
    name: "HelloWorld",
    props: {
        msg: String,
    },
    data() {
        return {
            playback: null,
            playback2: null,
            player: null,
            playbackStatus: "paused",
        };
    },

    mounted() {
        this.initializePeertubePlayer();
    },

    methods: {
        async initializePeertubePlayer() {
            let player = new PeerTubePlayer(document.getElementById("peertube"));

            let player2 = new PeerTubePlayer(document.getElementById("imlabs"));

            await player.ready;
            await player2.ready;

            player.addEventListener("playbackStatusUpdate", (playback) => {
                console.log("Chegou aqui dentro do event: ", playback);
                this.playback = playback;
            });

            player2.addEventListener("playbackStatusUpdate", (playback) => {
                console.log("Chegou aqui dentro do event: ", playback);
                this.playback2 = playback;
            });
        },
    },
};
</script>
```

https://github.com/Chocobozzz/PeerTube/assets/132066369/79bfb8c0-4db7-4e8d-99ce-5271339ed3b5

### Describe the expected behavior

We need the data to be updated to control the progression on our website, allowing it to advance only at the end of the video.

### Additional information

* PeerTube instance for docker: 
  * URL: https://a.video.imlabs.com.br/
  * Version: 6.0.1

Safari, only in iOS devices

**Comment by Chocobozzz** [comment]
Thanks! It should be fixed by https://github.com/Chocobozzz/PeerTube/commit/d05af753bf49f9e76b90825311ef007878164ca7

**Comment by Will-lopesz** [comment]
@Chocobozzz Thanks for the adjustment, do you plan to publish a new version with the correction?

**Comment by Chocobozzz** [comment]
The fix will be released in 6.2, in a few weeks/months. You can use [nightly builds](https://builds.joinpeertube.org/nightly/) if you don't want to wait :)
